docker/nginx/conf.d/include/ssl-settings

```diff
@@ -0,0 +1,13 @@
+# https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1d&hsts=false&ocsp=false&guideline=5.6
+
+ssl_session_timeout 1d;
+ssl_session_cache shared:MozSSL:10m;  # about 40000 sessions
+ssl_session_tickets off;
+
+# curl https://ssl-config.mozilla.org/ffdhe2048.txt > /path/to/dhparam
```

Comment:
Is this comment still valid? Aren't we generating a new one when we build the container?

Reply:
this whole file is a copy & paste from the generator - link is on top of the file https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1d&hsts=false&ocsp=false&guideline=5.6

I think it's beneficial to keep it unchanged as much as possible since we might want to periodically update this file based on the generator once we update nginx to support new tls protocols or ssl ciphers etc

We're not generating a new file on each server/build - we copied https://ssl-config.mozilla.org/ffdhe2048.txt once to our codebase and we're including it in Dockerfile

Reply:
I see. Well, if that's their recommended style then that's fine. Let's use their recommendation.

Reply:
1. That makes sense, let's keep it as-is.

2. Oh, right, this is the `dhparam` which is in its own file. What I mixed it up with was  this part of `docker/nginx/Dockerfile`:
```
    openssl req -new -newkey rsa:2048 -days 3650 -nodes -x509 \
    -subj '/CN=local-certificate' \
    -keyout /etc/ssl/local-certificate.key \
    -out /etc/ssl/local-certificate.crt
```